GitUI/CommandsDialogs/BrowseDialog/GitWorkingDirectoryStatusEventArgs.cs

```diff
@@ -14,7 +14,6 @@ public GitWorkingDirectoryStatusEventArgs(IEnumerable<GitItemStatus> itemStatuse
 
         public GitWorkingDirectoryStatusEventArgs()
         {
-            ItemStatuses = Enumerable.Empty<GitItemStatus>();
         }
 
         public IEnumerable<GitItemStatus> ItemStatuses { get; }
```

Comment:
If `null` now has a special meaning, please document that. Also include `[CanBeNull]` on this property.

Reply:
Done. Though in general, any pointer can be null unless it is attributed as [NotNull].

Reply:
Also, when we upgrade to C# 8 these annotations will help in the conversion to nullable reference types.